app/controllers/bike_share_app.rb

```diff
@@ -1,3 +1,158 @@
+require_relative '../models/station'
+require_relative '../helpers/bike_share_app_helper'
+
 class BikeShareApp < Sinatra::Base
 
+  include BikeShareAppHelper
+
+  set :root, File.expand_path("..", __dir__)
+  set :method_override, true
+
+  get '/' do
+    erb :landing_page
+  end
+
+  ### Start Stations Routes ###
+  get '/station-dashboard' do
+    @stations = Station.all
+    erb :"stations/dashboard"
```

Comment:
i like that you've nested your views in a "stations" directory to keep your views organized. good work 🎉